airbyte-integrations/connectors/source-rest-api/source_rest_api/source.py

```diff
@@ -45,7 +45,7 @@ def check(self, logger: AirbyteLogger, config_container) -> AirbyteConnectionSta
             return AirbyteConnectionStatus(status=Status.FAILED, message=r.text)
 
     def discover(self, logger: AirbyteLogger, config_container) -> AirbyteCatalog:
-        json_schema = {"$schema": "http://json-schema.org/draft-07/schema#", "additionalProperties": True, "type": "object"}
+        json_schema = {"$schema": "http://json-schema.org/draft-07/schema#", "additionalProperties": True, "type": "object", "properties": {}}
```

Comment:
discovery was breaking because AirbyteCatalogConverters expects everything to have a `properties` field

Reply:
shoudln't the converters be more lenient instead?

Reply:
yes, but I didn't want to change a core piece of the codebase. Would prefer to do this instead then fix the core pieces after release